From 900a4c853f8dbf0887c66155ba489ff4570e5948 Mon Sep 17 00:00:00 2001
From: Matthew Barnes <mbarnes@redhat.com>
Date: Wed, 01 Dec 2010 00:19:12 +0000
Subject: Bug 636100 - Can't read GSettings:backend property

The PROP_BACKEND case was missing from the switch statement in
g_settings_get_property().
---
diff --git a/gio/gsettings.c b/gio/gsettings.c
index 94d1f3a..7f775a0 100644
--- a/gio/gsettings.c
+++ b/gio/gsettings.c
@@ -417,6 +417,10 @@ g_settings_get_property (GObject    *object,
       g_value_set_string (value, settings->priv->schema_name);
       break;
 
+     case PROP_BACKEND:
+      g_value_set_object (value, settings->priv->backend);
+      break;
+
      case PROP_PATH:
       g_value_set_string (value, settings->priv->path);
       break;
--
cgit v0.8.3.1
